srcpkgs/hunspell-ca_ES/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'hunspell-ca_ES'
+pkgname=hunspell-ca_ES
+version=3.0.6
+revision=1
+create_wrksrc=yes
+hostmakedepends="unzip"
+short_desc="Catalan dictionary for hunspell"
+maintainer="Orphaned <orphan@voidlinux.org>"
```

Comment:
this should be set to your name or username and a valid email that you can be reached at

Reply:
I'm on it. Once I clean up the mess I made by clicking on github's "fetch upstream" I'll --force commit.
Sorry for the spam.